modules/swagger-codegen/src/main/resources/nodejs-google-cloud-functions/index.mustache

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
I found that this file is the same as `index-gcf.mustache` so I simply removed it via https://github.com/swagger-api/swagger-codegen/pull/4808. I did some tests and the generator is still working properly with or without the option for Google cloud function.